firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/persistence/CrashlyticsReportPersistence.java

```diff
@@ -151,22 +178,55 @@ public void finalizeReports(String currentSessionId) {
   }
 
   public List<CrashlyticsReport> loadFinalizedReports() {
-    final List<CrashlyticsReport> reports = new ArrayList<>();
-    final List<File> fatalReports = getAllFilesInDirectory(fatalReportsDirectory);
-    for (File reportFile : fatalReports) {
-      reports.add(TRANSFORM.reportFromJson(readTextFile(reportFile)));
+    final List<CrashlyticsReport> allReports = new ArrayList<>();
+    final List<File> priorityReports = getAllFilesInDirectory(priorityReportsDirectory);
+    for (File reportFile : priorityReports) {
+      allReports.add(TRANSFORM.reportFromJson(readTextFile(reportFile)));
     }
-    final List<File> nonFatalReports = getAllFilesInDirectory(nonFatalReportsDirectory);
-    for (File reportFile : nonFatalReports) {
-      reports.add(TRANSFORM.reportFromJson(readTextFile(reportFile)));
+    final List<File> reports = getAllFilesInDirectory(reportsDirectory);
+    for (File reportFile : reports) {
+      allReports.add(TRANSFORM.reportFromJson(readTextFile(reportFile)));
     }
-    return reports;
+    return allReports;
+  }
+
+  private static boolean isHighPriorityEvent(String eventFileName) {
+    return eventFileName.endsWith(PRIORITY_EVENT_SUFFIX);
+  }
+
+  private String generateEventFilename(boolean isHighPriority) {
+    final String eventNumber =
+        String.format(Locale.US, EVENT_COUNTER_FORMAT, eventCounter.getAndIncrement());
```

Comment:
It is interesting to me that this method is side-effecting and returns a value. Would it be more obvious to the reader if this method were static, and you passed  `generateEventFilename(eventCounter.getAndIncrement(), isHighPriority)` in?

Reply:
(I guess you'd probably want to rename it if you did that. I could go either way really, just wanted to ask!)

Reply:
Solid point. I don't think it would neccessitate a rename.